lib/opte/src/engine/ip6.rs

```diff
@@ -637,7 +648,7 @@ pub(crate) mod test {
                     let mut packet =
                         Ipv6ExtHeader::new_checked(&mut buf).unwrap();
                     packet.set_next_header(IpProtocol::Tcp);
-                    packet.set_header_len(((route.buffer_len() - 6) / 8) as u8);
+                    packet.set_header_len((route.buffer_len() / 8) as u8);
```

Comment:
With integer math it works out the same, but the prev was exact. Was the change to save on more magic numbers? (it is a pain how it seems there's a lot more of this manual  with every smoltcp update 😞)

Reply:
That's effectively what I was going for, particularly since that line needed to be duplicated -- I'll add in a comment at least clarifying the logic around 8-octet blocks. Agreed on the smoltcp-mandated changes.